build/websocket/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM --platform=$BUILDPLATFORM docker.io/golang:1.18 as builder
+WORKDIR /go/src
+# Copy `go.mod` for definitions and `go.sum` to invalidate the next layer
+# in case of a change in the dependencies
+COPY ../server/go.mod ../server/go.sum ./
+# Download dependencies
+RUN go mod download
+
+ARG TELEMETRY_TOKEN
+ARG VERSION
+ARG TARGETARCH
+ARG TARGETOS
+
+COPY ../server ./
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_ENABLED=0 go build -v -ldflags "-X github.com/kubeshop/kusk-gateway/pkg/analytics.TelemetryToken=$TELEMETRY_TOKEN -X github.com/kubeshop/kusk-gateway/pkg/build.Version=$VERSION" -o kusk-gateway-api-websocket cmd/websocket/*.go
```

Comment:
running `go build` just on main returns this output
```
➜ go build main.go                       
# command-line-arguments
./main.go:38:20: undefined: NewClientSet
./main.go:44:22: undefined: websocketHandler
```

This is because all the code is in the same `websocket` package. It would work if everything `main.go` depended on lived in different packages that were then imported in main

Reply:
OK. Resolving.